content/en/docs/setup/install/virtual-machine/common.sh

```diff
@@ -69,7 +73,7 @@ function setup_vm() {
     snip_configure_the_virtual_machine_1
     snip_configure_the_virtual_machine_2
     # TODO: we should probably have a better way to get the debian package
-    curl -LO https://storage.googleapis.com/istio-build/dev/1.9-alpha.cdae086ca8cae8be174c8feee509841f89792e43/deb/istio-sidecar.deb
+    curl -LO https://storage.googleapis.com/istio-build/dev/${DEV_VERSION}/deb/istio-sidecar.deb
```

Comment:
Maybe we should have this use the version calculated in init? Built from VERSION and commit.

Reply:
Updated, thanks!